build.js

```diff
@@ -62,6 +66,25 @@ const sitemapOpts = {
   ],
 };
 
+// --- FORMAT
+const loadEmoji = () =>
+  fetch('https://api.github.com/emojis')
+    .then(r => r.json())
+    .catch(handleFailure);
+
+let emojiMapURL = {};
+
+const emojify = text => {
+  if (!text) return text;
+  const colonWrapped = /(:[\w\-+]+:)/g;
+  const result = text.replace(colonWrapped, match => {
+    const name = match.replace(/:/g, '');
+    const url = emojiMapURL[name];
+    return `<img src="${url}" class="emoji" alt="${name}" />`;
```

Comment:
`url` may be `null` if map doesn't contain key/emoji

Reply:
good catch 👍